templates/.github/workflows/go-check.yml

```diff
@@ -48,4 +56,17 @@ jobs:
           run: |
             set -o pipefail
             staticcheck ./... | sed -e 's@\(.*\)\.go@./\1.go@g'
+      - name: go generate
+        uses: protocol/multiple-go-modules@v1.2
+        if: env.RUNGOGENERATE == 'true'
+        with:
+          run: |
+            git clean -fd # make sure there aren't untracked files / directories
+            go generate ./...
+            gitstatus=$(git status --porcelain)
+            if [[ "$gitstatus" != "" ]]; then
```

Comment:
`if [[ -z "$gitstatus" ]]`

Reply:
👍🏻

Reply:
Actually, this has to be `-n`, not `-z`.